modules/integration/src/test/scala/scala/cli/integration/RunTestDefinitions.scala

```diff
@@ -211,7 +230,9 @@ abstract class RunTestDefinitions(val scalaVersionOpt: Option[String])
       )
     )
     inputs.fromRoot { root =>
-      val output = os.proc(TestUtil.cli, extraOptions, "dir", "--main-class", "print").call(cwd =
+      // TODO: It's probably impossible to handle `--main-class print`, because we put main method in the other
```

Comment:
@alexarchambault @lwronski I couldn't find any way to fix it. This means the scripts users will have to pass `--main-class <somethnig>_sc`, or we will have to add `_sc` suffix silently in options engine